PlayCover/Views/MainView.swift

```diff
@@ -38,6 +42,45 @@ struct MainView: View {
                                                                        selectedTextColor: $selectedTextColor)
                         } label: {
                             Label("sidebar.appLibrary", systemImage: "square.grid.2x2")
+                            Button {
+                                withAnimation {
+                                    showAppFolders.toggle()
+                                }
+                            } label: {
+                                Image(systemName: showAppFolders ? "chevron.up" : "chevron.down")
+                                    .font(.caption)
+                            }
+                            .buttonStyle(.plain)
+                                .contextMenu(menuItems: {
```

Comment:
This should be tabbed one in for syntax, and theres an extra newline at the end of the context menu.